ygnmi/unmarshal.go

```diff
@@ -47,6 +47,21 @@ type DataPoint struct {
 	Sync bool
 }
 
+func (d *DataPoint) String() string {
+	if d == nil {
+		return ""
+	}
+	path, err := ygot.PathToString(d.Path)
+	if err != nil {
+		path = prototext.Format(d.Path)
+	}
+	valStr := fmt.Sprint(d.Value)
+	if jsonietf := d.Value.GetJsonIetfVal(); len(jsonietf) > 0 {
+		valStr = string(jsonietf)
+	}
+	return fmt.Sprintf("%s (timestamp: %v, recvTimestamp: %v, isSync: %v):\n%s", path, d.Timestamp, d.RecvTimestamp, d.Sync, valStr)
```

Comment:
nit: when these show up in logs i usually prefer that they on one line. maybe don't include the newline just for non-json types?

Reply:
Just removed the newline since for JSON there is either a { or not, and in both cases not having a newline makes sense.